addons/website_sale/views/website_pages_views.xml

```diff
@@ -31,13 +31,36 @@
             </field>
             <xpath expr="//tree">
                 <field name="is_seo_optimized"/>
-                <field name="is_published"/>
-
+                <field name="publish_on" column_invisible="1"/>
+                <field name="website_published" invisible="publish_on and website_published"/>
+                <field name="publish_on" string="Scheduled" invisible="not publish_on"/>
                 <field name="website_id" position="move"/>
             </xpath>
         </field>
     </record>
-
+    <record id="product_pages_form_view" model="ir.ui.view">
+        <field name="name">Product pages form view</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="arch" type="xml">
+            <notebook position="inside">
+                <page name="publishing" string="Publish">
+                    <group>
+                        <field name="publish_on" invisible="1"/>
+                        <field string="Published" name="website_published" widget="boolean_toggle" readonly="publish_on"></field>
+                        <label invisible="website_published" for="publish_on" string="Auto publish on"/>
+                        <div invisible="website_published" class="o_row">
+                            <field name="publish_on" widget="datetime" options="{'min_date': datetime.datetime.now().to_utc().strftime('%Y-%m-%d %H:%M:%S')}"/>
+                            <button name="action_unschedule" type="object" invisible="not publish_on">Unschedule</button>
+                        </div>
+                    </group>
+                    <div class="alert alert-info w-100" invisible="not publish_on" role="alert">
+                        <small>This product will automatically go live on the specified date</small>
+                    </div>
+                </page>
+            </notebook>
+        </field>
+    </record>
```

Comment:
Why don't you include this data in the existing `product_template_form_view` in `website_sale/views/product_views.xml` ?

Reply:
@Feyensv Good question :) ... It's been moved.

Reply:
To the file I was speaking about, yes.
But why do you need a specific view for that content, there is already a view inheriting `product_template_only_form_view` (`product_template_only_website_form_view`).
AFAICS, your view isn't primary, so I don't see a reason to keep it in a separate record.

Reply:
@Feyensv You're absolutely right. It's done.